pkg/kubectl/resource/builder.go

```diff
@@ -256,6 +256,23 @@ func (b *Builder) SelectAllParam(selectAll bool) *Builder {
 // When two or more arguments are received, they must be a single type and resource name(s).
 // The allowEmptySelector permits to select all the resources (via Everything func).
 func (b *Builder) ResourceTypeOrNameArgs(allowEmptySelector bool, args ...string) *Builder {
```

Comment:
Can you add a test case in resource/builder_test.go to verify the new cases we've added (`1,2,3 a b` and `1,2 a`)?